fathom/src/surface/elaboration/reporting.rs

```diff
@@ -87,10 +87,9 @@ pub enum Message {
     /// Unification errors.
     FailedToUnify {
         range: ByteRange,
-        // TODO: add lhs and rhs values
-        // lhs: Doc<_>,
-        // rhs: Doc<_>,
-        error: unification::Error,
+        lhs: String,
+        rhs: String,
+        error: unification::Error<'core>,
```

Comment:
I think I had envisioned translating to a surface level unification error, seeing as value can't really be nicely rendered without the names that were bound at the point where the values were unified. But yeah, that might be pointless busywork, not sure.